src/lib/oauth2-service.ts

```diff
@@ -20,11 +20,13 @@
  */
 
 import { IncomingMessage } from 'http';
+import { URL } from 'url';
```

Comment:
@poveden With the latest update, TypeScript started to complain that it didn't know about that type. Had to reference it explicitly